packages/xo-server-load-balancer/src/performance-plan.js

```diff
@@ -70,12 +70,18 @@ export default class PerformancePlan extends Plan {
 
       debug(`Try to optimize Host (${exceededHost.id}).`)
       const availableHosts = filter(hosts, host => host.id !== id)
+      debug(
+        `Available destinations: ${mapToArray(
+          availableHosts,
+          host => host.id
+        )}.`
+      )
 
       // Search bests combinations for the worst host.
       await this._optimize({
         exceededHost,
         hosts: availableHosts,
-        hostsAverages: averages,
+        hostsAverages: averages
```

Comment:
Weird change, either revert or make sure you are up to date with `master` and that all dependencies are up to date (`yarn`).